Assets/MRTK/Providers/Oculus/XRSDK/Controllers/OculusXRSDKTouchController.cs

```diff
@@ -108,9 +108,20 @@ protected override bool TryRenderControllerModel(System.Type controllerType, Inp
             }
         }
 
-        private async void TryRenderControllerModelFromOculus()
+        private void TryRenderControllerModelFromOculus()
         {
-            await WaitForOculusVisuals();
+            OculusXRSDKDeviceManager deviceManager = CoreServices.GetInputSystemDataProvider<OculusXRSDKDeviceManager>();
```

Comment:
Just to be safe, we should probably null-check the result here

Reply:
Done!